ceph.spec.in

```diff
@@ -1266,6 +1317,12 @@ fi
 %{python_sitearch}/rados-*.egg-info
 
 #################################################################################
+%files -n python%{python3_pkgversion}-rados
+%defattr(-,root,root,-)
+%{python3_sitearch}/rados.cpython*.so
+%{python3_sitearch}/rados-*.egg-info
+
+#################################################################################
```

Comment:
the `#######` divider lines are being dropped in #9757, so can you remove them from your patch (just the new ones that you're adding)